website/releases/release-0.14.0.md

```diff
@@ -0,0 +1,319 @@
+link) and Spark 3.0 (hudi-spark3.0-bundle_2.12) (TODO Add link). Please note that, the support for Spark 3.0 had been 
+discontinued after Hudi version 0.10.1, but due to strong community interest, it has been reinstated in this release.
+
+### Breaking Changes
+
+#### INSERT INTO behavior with spark-sql
+Before version 0.14.0, data ingested through `INSERT INTO` in Spark SQL followed the upsert flow, where multiple versions 
+of records would be merged into one version. However, starting from 0.14.0, we've altered the default behavior of 
+`INSERT INTO` to utilize the `insert` flow internally. This change significantly enhances write performance as it 
+bypasses index lookups.
+
+If a table is created with a *preCombine* key, the default operation for `INSERT INTO` remains as `upsert`. Conversely, 
+if no *preCombine* key is set, the underlying write operation for `INSERT INTO` defaults to `insert`. Users have the 
+flexibility to override this behavior by explicitly setting values for the config 
+[`hoodie.spark.sql.insert.into.operation`](https://hudi.apache.org/docs/configurations#hoodiesparksqlinsertintooperation) 
+as per their requirements. Possible values for this config include `insert`, `bulk_insert`, and `upsert`.
+
+Additionally, in version 0.14.0, we have **deprecated** two related older configs:
+- `hoodie.sql.insert.mode`
+- `hoodie.sql.bulk.insert.enable`.
+
+### Behavior changes
+
+#### Simplified duplicates handling with Inserts
+In cases where the operation type is configured as `insert`, users now have the option to enforce a duplicate policy 
+using the configuration setting 
+[`hoodie.datasource.insert.dup.policy`](https://hudi.apache.org/docs/configurations#hoodiedatasourceinsertduppolicy). 
+This policy determines the action taken when incoming records being ingested already exist in storage. The available 
+values for this configuration are as follows:
+
+- `none`: No specific action is taken, allowing duplicates to exist in the Hudi table if the incoming records contain duplicates.
+- `drop`: Matching records from the incoming writes will be dropped, and the remaining ones will be ingested.
+- `fail`: The write operation will fail if the same records are re-ingested. In essence, a given record, as determined 
+by the key generation policy, can only be ingested once into the target table.
+
+With this addition, an older related configuration setting, 
+[`hoodie.datasource.write.insert.drop.duplicates`](https://hudi.apache.org/docs/configurations#hoodiedatasourcewriteinsertdropduplicates), 
+is now deprecated. The newer configuration will take precedence over the old one when both are specified. If no specific 
+configurations are provided, the default value for the newer configuration will be assumed. Users are strongly encouraged 
+to migrate to the use of these newer configurations. (TODO: Verify this behavior)
```

Comment:
I think the new behavior was well tested, but you can confirm with @jonvex or @ad1happy2go